src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -400,3 +400,57 @@ def test_acr_create_build_task(self, resource_group, resource_group_location):
 
         # test acr delete
         self.cmd('acr delete -n {registry_name} -g {rg}')
+
+    @ResourceGroupPreparer()
+    def test_acr_image_import(self, resource_group):
+        '''There are six test cases in the function. To be able to run the tests, we are having the following resources before the test:
+        Current cloud and active subscription: AzureCloud ACR - Dev
+        Two source registries, one in the subscription other than the ACR - Dev and another in the subscription as the current subsciption.
+        Two source images each of which stays in a different source registries.
+        '''
+
+        registry_name = self.create_random_name("targetregsitry", 20)
+
+        self.kwargs.update({
+            'registry_name': registry_name,
+            'rg_loc': 'eastus',
+            'sku': 'Standard',
+            'r_id': '/subscriptions/a7ee80a4-3d5e-45c2-9378-36e8d98f4d13/resourceGroups/yugongTest2/providers/Microsoft.ContainerRegistry/registries/yugongTest2',
```

Comment:
I would recommend moving the above description here, mentioning this registry and below images were already prepared.
NIT: use some generic names for the resource group and registry, e.g., sourceRegistryGroup, sourceRegistry (not sure if this one still exists), etc.